.github/.jira_sync_config.yaml

```diff
@@ -0,0 +1,23 @@
+# Sync GitHub issues to Jira issues
+
+# Configuration syntax:
+# https://github.com/canonical/gh-jira-sync-bot/blob/main/README.md#client-side-configuration
+settings:
+  # Repository specific settings
+  components: # Jira components that will be added to Jira issue
+    - opensearch-vm
+    - opensearch-k8s
```

Comment:
@Mehdi-Bendriss Should these components be `opensearch-dashboards-vm` and `opensearch-dashboard-k8s`?

Reply:
IMHO, yes.

Reply:
Discussed in private with @carlcsaposs-canonical 
Keeping AS IS, as we are moving the current logic. 
Carl will notify @Mehdi-Bendriss about the wrong label here to update separately avoiding Jira Dashboards tickets loose.